### ISSUE TITLE
Trouble installing pycoQC

### ISSUE THREAD

**Issue by laulambr** [a-slide/pycoQC#74]
**Describe the bug**
Hi I am having trouble installing pycoQC.
I followed the instructions and created a venv running Python 3.5.2. 
I installed using Option 1: Installation with pip from pypi.
After the installation was completed and I tried to run pycoQC I got the following error:
```

Traceback (most recent call last):
  File "/data/laurensl/venv/bin/pycoQC", line 6, in <module>
    from pycoQC.__main__ import main_pycoQC
  File "/data/laurensl/venv/lib/python3.5/site-packages/pycoQC/__main__.py", line 18, in <module>
    from pycoQC.pycoQC import pycoQC
  File "/data/laurensl/venv/lib/python3.5/site-packages/pycoQC/pycoQC.py", line 13, in <module>
    from pycoQC.pycoQC_plot import pycoQC_plot
  File "/data/laurensl/venv/lib/python3.5/site-packages/pycoQC/pycoQC_plot.py", line 179
    height:int=300+(30*self.all_df[groupby].nunique()) if groupby else 300
          ^
SyntaxError: invalid syntax
```

How can I fix this?

**Comment by a-slide** [comment]
Hi Laulambr

I think I fixed it in the last release (2.3.1.3). I works for me, but let me know if you have any issues with it.
(Sorry for the delay I was on holiday).

**Comment by laulambr** [comment]
Hi

I just reinstalled and it seems to be working right now. Thank you for taking the time to respond!
